addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -368,8 +368,14 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
         const proms = [this._super(...arguments)];
         // Disable text edition
         this.$target.attr('contentEditable', false);
-        // Make button, description, and recaptcha editable
-        this.$target.find('.s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha').attr('contentEditable', true);
+        const formEditableColSelector = ".row > div:not(.s_website_form_field, .s_website_form_submit):not(.s_website_form_field *):not(.s_website_form_submit *)";
+        // Identify editable elements of the form: buttons, description,
+        // recaptcha and columns which are not fields.
+        const formEditableSelector = `${formEditableColSelector}, .s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha`
+            .split(", ").map(selector => `:scope ${selector}`).join(", ");
```

Comment:
I don't think you want that on in the middle of a `:not`: 
![image](https://github.com/odoo/odoo/assets/10338094/af4d7970-ab63-40b6-a7c1-d54078c8a64d)

Reply:
Good catch! Updated now :+1: